src/tasks/AndroidAppBuilder/ApkBuilder.cs

```diff
@@ -313,6 +311,21 @@ public ApkBuilder(TaskLoggingHelper logger)
             }
         }
 
+        if (StaticLinkedRuntime && IsCoreCLR)
+        {
+            string[] staticLibs = Directory.GetFiles(AppDir, "*.a")
+                .Where(lib => !Path.GetFileName(lib).Equals("libcoreclr_static.a", StringComparison.OrdinalIgnoreCase))
```

Comment:
Why are we excluding `libcoreclr_static.a` in this case? That's the one I would expect we want to keep for sure...

Reply:
It was included before:
```csharp
else if (StaticLinkedRuntime && IsCoreCLR)
{
    runtimeLib = Path.Combine(AppDir, "libcoreclr_static.a");
}

if (!File.Exists(runtimeLib))
{
    throw new ArgumentException($"{runtimeLib} was not found");
}
else
{
    nativeLibraries += $"{runtimeLib}{Environment.NewLine}";
}
```

At this stage, other libraries are being collected. If `libcoreclr_static.a` is included (again), it may result in duplicate symbols.